charts/karpenter/values.yaml

```diff
@@ -78,8 +78,12 @@ extraVolumes: []
 #         path: token
 
 controller:
-  # -- Controller image.
-  image: "public.ecr.aws/karpenter/controller:v0.10.0@sha256:e27cc9fb91f80ed9c5c26202c984a9d7d871ce6008dd4f83f50f3516c9f2ce8e"
+  image:
+    repository: public.ecr.aws/karpenter/controller
+    # -- Defaults to chart.appVersion, skip with "-"
+    tag:
+    # -- sha256:xxx, can be combined with tag
+    digest:
```

Comment:
I suspect that the maintainers will want the digest kept.

Reply:
If the maintainers want to keep using the digest; there is not much added value in this PR. I think it's better to have a split into repo and tag and that's it. Thoughts?

Reply:
I think this PR adds value even if the maintainers want to keep specifying the digest. Personally I think the chart should either have an immutable image with digest or capturing the digest should be a separate concern to the chart.